.travis.yml

```diff
@@ -16,6 +16,7 @@ before_script:
 script:
 - make test-all
 - make test-js
+- make validate_translations
```

Comment:
Nit: It might be better to run this on line 17, before test-all and test-js. I think it would be better to find out about translation issues up front, rather than having to wait for all the tests to finish.